docs/policies/policies.md

```diff
@@ -0,0 +1,515 @@
+
+## General Information about Policies
+
+ℹ️ All explanations in this chapter "General Information about Policies" were taken from the following [source](https://w3c.github.io/poe/model/).
+
+### Introduction
+
+Several business scenarios require expressing what are the permitted and prohibited actions over resources. These permitted/prohibited actions are usually expressed under the form of policies, i.e., expressions that indicate those uses and re-uses of the content which are conformant with existing regulations or to the constraints assigned by the owner. Policies may also be enriched with additional information, i.e., who are the entities in charge of the definition of such Policy and those who are required to conform to it, what are the additional constrains to be associated with the Permissions, Prohibitions and Duties expressed by the Policy. The ability to express these concepts and relationships is important both for the producers of content, i.e., they may state in a clear way what are the permitted and the prohibited actions to prevent misuse, and for the consumers, i.e., they may know precisely what resources they are allowed to use and re-use to avoid breaking any rules, laws or the owner's constraints. This specification describes a common approach to expressing these policy concepts.
+
+### Semantic Model
+
+The ODRL Information Model defines the underlying semantic model for permission, prohibition, and obligation statements describing content usage. The information model covers the core concepts, entities and relationships that provide the foundational model for content usage statements. These machine-readable policies may be linked directly with the content they are associated to with the aim to allow consumers to easily retrieve this information.
+
+#### Most Important Terminology
+
+| Term     | Description |
+| -------- | -------- |
+| Policy | A group of one or more Rules
+| Rule | An abstract concept that represents the common characteristics of Permissions, Prohibitions, and Duties
+| Asset | A rsource or a collection of resources that are the subject of a Rule
+| Action | An operation on an Asset
+| Permission | The ability to exercise an Action over an Asset
+| Prohibition | The inability to exercise an Action over an Asset
+| Constraint | A boolean/logical expression that refines an Action and Asset collection or the conditions applicable to a Rule.
+
+## Policies in Catena-X
+
+A Catena-X policy uses the ODRL Information Model as a semantic model. Theoretically, a policy can be constructed individually using this ODRL Information Model. Catena-X does not yet use all the possibilities that ODRL provides, so the next example will show how a Catena-X policy is basically structured.
+
+### Policy Template Example with @context: `"http://www.w3.org/ns/odrl.jsonld"`
+
+```json
+{
+  "@context": {},
+  "@type": "PolicyDefinitionRequest",
+  "@id": "{{POLICY_ID}}",
+  "policy": {
+    "@type": "Set",

```

Comment:
The `@type` is sometimes set to `Set`, sometimes to `odrl:Set`, and sometimes to `Policy`. This should be updated and aligned.

Reply:
"The above example could have also used Policy type instead of Set type (**as they are equivalent**)." [Source](https://w3c.github.io/poe/model/#policy-set)

`odrl:set `is also the same. Just defined in another `@context`